roles/lnls-ans-role-sirius-hlacon/tasks/remove-hlacon-from-system.yml

```diff
@@ -22,7 +22,9 @@
   pip:
     name: "{{ sirius_apps_hlacon_pip_packages_to_remove }}"
     state: absent
-    executable: "{{ pip_executable | default(omit) }}"
+    executable: "{{ conda_python_pip_executable | default(omit) }}"
+  vars:
+    ansible_python_interpreter: "{{ conda_python_executable }}"
```

Comment:
wouldn't it be `ansible_python_interpreter: "{{ conda_python_pip_executable }}"`
like in https://github.com/lnls-sirius/lnls-ansible/pull/364/commits/f64dd9001c705049bb2124d6f9a582ed8246640a#diff-960dcb6f7eafb6e80ba5ec2b88bf47c4afa04a8660fd404b695763143d4f5e8eR28